main.py

```diff
@@ -175,15 +199,27 @@ def __init__(self, master):
         self.imageIdxLabel = Label(self.statusBar, text="                      ")
         self.imageIdxLabel.pack(side="right", fill=X)
 
+    def get_session(self):
+        config = tf.ConfigProto()
+        config.gpu_options.allow_growth = True
+        return tf.Session(config=config)
+
+    def available_models(self):
+        self.models_dir = os.path.join(cur_path, 'snapshots')
+        # only for keras and tf
+        model_categ = os.listdir(self.models_dir)
+        # creating all model options list
+        model_names = []
+        for categ in model_categ:
+            for name in os.listdir(os.path.join(self.models_dir , categ)):
+                model_names.append(os.path.join(categ,name))
```

Comment:
After adding the README.md file in the snapshots directory, the listdir for model_categ also returns README.md and the iterator cannot then list it further as it is a file.

Changing line 210,
From:
`model_categ = os.listdir(self.models_dir) `
To:
`model_categ = [dir_ for dir_ in os.listdir(self.models_dir) if os.path.isdir(os.path.join(self.models_dir, dir_))]`

should do the trick.

Reply:
Yeah... I also noticed it now only...thanks